staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -917,9 +917,9 @@ const (
 // +k8s:openapi-gen=true
 type VirtualMachineStatus struct {
 	// Created indicates if the virtual machine is created in the cluster
-	Created bool `json:"created,omitempty"`
+	Created bool `json:"created"`
 	// Ready indicates if the virtual machine is running and ready
-	Ready bool `json:"ready,omitempty"`
+	Ready bool `json:"ready"`
```

Comment:
ah, so by removing omitempty here, it is being inferred that this boolean is now a required field (which it shouldn't be). 

can we leave omitempty?

Reply:
@davidvossel Yes, but until VMI will be created the status won't exist in the VM resource and then the column will be empty.